src/test/scala/it/pagopa/interop/backendforfrontend/AuthorizationApiServiceSpec.scala

```diff
@@ -26,12 +30,12 @@ class AuthorizationApiServiceSpec extends AnyWordSpecLike with SpecHelper with S
     "succeed" in {
```

Comment:
We need an end to end test for this api that sends the claims encoded through tcp to be sure we won't have problems again